indexer/metadata/simulation/iceprod_tools.py

```diff
@@ -1,10 +1,12 @@
+# iceprod_tools.py
```

Comment:
Curious, why is this here?

Reply:
Just an old habit. I like to put the name of the file in a comment at the top.
I think it comes from adding the GPL stanza. :smile: